### PR TITLE
feat(48 > main): add link back & front to login & register

### PR DESCRIPTION
**Pull request by AlexGuillard**

### CODE REVIEW

**Review by Pablodeibar — APPROVED** [review]
looks good

File: web/src/Page/Register/register.tsx [hunk 1]
```diff
@@ -15,6 +19,24 @@ function Register() {
...
 
+  const handleClickRegister = () => {
+    const data = {
+      mail: textEmail,
+      password: textPassWord,
+    };
+    console.log(process.env.REACT_APP_SERVER_URL)
+    axios.post(process.env.REACT_APP_SERVER_URL + '/auth/signup', data)
+      .then(response => {
+        console.log(response.data)
```
Comment (Benjicatch) [inline web/src/Page/Register/register.tsx:30]: remove console.log

File: web/src/Page/Register/register.tsx [hunk 2]
```diff
@@ -15,6 +19,24 @@ function Register() {
       setTextPassWord(event.target.value);
   };
 
+  const handleClickRegister = () => {
+    const data = {
+      mail: textEmail,
+      password: textPassWord,
+    };
+    console.log(process.env.REACT_APP_SERVER_URL)
```
Comment (Benjicatch) [inline web/src/Page/Register/register.tsx:27]: remove

**Review by Benjicatch — CHANGES_REQUESTED** [review]

**Review by Pablodeibar — APPROVED** [review]
looks good

**Review by Benjicatch — APPROVED** [review]
ok

**Pull request merged**